packages/web/src/sideEffects/http/index.js

```diff
@@ -1,43 +1,35 @@
-// const most = require('most')
+const url = require('url')
+const path = require('path')
+
 const callBackToStream = require('@jscad/core/observable-utils/callbackToObservable')
 const makeLogger = require('../../utils/logger')
 const getFileExtensionFromString = require('@jscad/core/utils/getFileExtensionFromString')
 
 const XMLHttpRequest = window.XMLHttpRequest
 
-/** function to create the http side effect
- * sink: input of commands that generate http requests
- * source: output of response from http requests
+// const proxyUrl = './remote.php?url='
+const proxyUrl = './remote.pl?url='
```

Comment:
Why .pl instead .php?  Just curious.

Reply:
Both Perl and PHP versions are available.

The Perl version is slightly stronger on security, and the webserver is configured to run CGI Perl scripts. So, for now, let's use the Perl version.

I may make some changes to the Perl script as well, as there are more supported formats now. But this will be another set of changes.